pkg/redist/System.Collections.Concurrent/pkg/System.Collections.Concurrent.pkgproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- bump up the version since we are redisting the 4.0.10 bits in a new package -->
```

Comment:
Is this comment correct? I'm not sure I'm following the logic here, I though this is the new package for the 4.0.10 bits.